package.json

```diff
@@ -31,7 +31,8 @@
         "react-window": "^1.8.5",
         "reconnecting-websocket": "^4.4.0",
         "redux": "^4.0.5",
-        "typeface-roboto": "0.0.75"
+        "typeface-roboto": "0.0.75",
+        "color-name": "latest"
```

Comment:
I don't think this is used. And even if you do use it, I don't think we need this dependency at all